app/main/forms/frameworks.py

```diff
@@ -34,3 +34,17 @@ class AcceptAgreementVariationForm(Form):
             DataRequired(message="You can only save and continue if you agree to the proposed changes.")
         ]
     )
+
+
+class ReuseDeclarationForm(Form):
+    """Form for the reuse declaration page. One yes no question.
+
+    `reuse` is a yes no whether they want to reuse a framework.
+    `old_framework` is a hidden field allowing us to pass back the framework slug of the framework they are choosing to
+    reuse.
+    """
+    reuse = BooleanField(
+        'Do you want to reuse the answers from your earlier declaration?',
+        validators=[InputRequired()]
```

Comment:
So I spoke to @laurenceberry about this and he said use the default (although he may not have know what the default was) I'd rather not let this block the PR given I've already asked for confirmation on this.